Production/python/scenarios.py

```diff
@@ -60,6 +60,13 @@ def __init__(self,sname):
                           jecfile="data/jec/Summer16_23Sep2016AllV3_DATA",
                           residual=True,
                           era="Run2_25ns")
+        elif sname == "2017PromptCollisions":
+            self.set_vars(globaltag="92X_dataRun2_Prompt_v8",
+                          tagname="RECO",
+                          jsonfile="data/Cert_294927-299042_13TeV_PromptReco_Collisions17_JSON.txt",
+                          jecfile="data/jec/Summer16_23Sep2016AllV3_DATA",
+                          residual=True,
+                          era="Run2_25ns")

```

Comment:
I added the new JSON: https://cms-service-dqm.web.cern.ch/cms-service-dqm/CAF/certification/Collisions17/13TeV/PromptReco/Cert_294927-299649_13TeV_PromptReco_Collisions17_JSON.txt